### PR TITLE
Comment body is just a string in the Jira v2 API

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Relates to #56

### CODE REVIEW

File: source/Server.Tests/WorkItemLinkMapperScenarios.cs [hunk 1]
```diff
@@ -46,21 +46,7 @@ public string GetWorkItemDescription(string linkData, string releaseNotePrefix,
...
-                                            {
-                                                new JiraDocContentElement
-                                                {
-                                                    Text = releaseNote
-                                                }
-                                            }
-                                        }
-                                    }
-                                }
+                                Body = releaseNote
```
Comment (hnrkndrssn) [inline source/Server.Tests/WorkItemLinkMapperScenarios.cs:49]: This is the main change related to the bug that customers are seeing

File: source/Server/Integration/JiraRestClient.cs [hunk 2]
```diff
@@ -150,33 +150,7 @@ class JiraIssueComments
...
-        public IEnumerable<JiraDocContentElement> Content { get; set; } = new JiraDocContentElement[0];
-    }
-
-    class JiraDocContentElement
-    {
-        [JsonProperty("type")]
-        public string Type { get; set; } = string.Empty;
-        [JsonProperty("text")]
-        public string Text { get; set; } = string.Empty;
+        public string? Body { get; set; }
```
Comment (hnrkndrssn) [inline source/Server/Integration/JiraRestClient.cs:153]: These too

File: source/Server/WorkItems/WorkItemLinkMapper.cs [hunk 3]
```diff
@@ -95,7 +95,7 @@ public string GetReleaseNote(JiraIssue issue, string? releaseNotePrefix)
 
             var releaseNoteRegex = new Regex($"^{releaseNotePrefix}", RegexOptions.Compiled | RegexOptions.IgnoreCase);
 
-            var releaseNote = issue.Fields.Comments.Comments.SelectMany(x => x.Body?.Content.SelectMany(b => b.Content).Select(x => x.Text)).Where(x => !(x is null)).LastOrDefault(c => releaseNoteRegex.IsMatch(c));
+            var releaseNote = issue.Fields.Comments.Comments.Select(x => x.Body).Where(x => x is not null).LastOrDefault(c => releaseNoteRegex.IsMatch(c));
```
Comment (hnrkndrssn) [inline source/Server/WorkItems/WorkItemLinkMapper.cs:98]: And this

**Review by johnsimons — APPROVED** [review]
LGTM

**Pull request merged**